dev/com.ibm.ws.jdbc_fat_oracle/publish/files/oracle-ssl/release.sh

```diff
@@ -4,10 +4,10 @@
 USER_NAME=kyleaure
 
 #Version of docker image.  Increment if doing a new release
-VERSION=2.0
+VERSION=1.0.full.ssl
```

Comment:
Since my previous images were built from scratch and these images are extended from `gvenzl/oracle-xe` I decided to put these in a different repository. 
Old Repo: kyleaure/oracle-prebuilt (because the database was prebuilt to improve performance over what was widely available.
New Repo: kyleaure/oracle-expanded (because the databas is uncompressed to improve performance over the `gvenzl/oracle-xe` base image)

Since its a new repo I started at v1.0